### PR TITLE
ENG-73 Update unicorn to get model selection cluster params from nupic.

### PR DESCRIPTION
**Pull request by vitaly-krugl**
WAIT! Don't merge this unit https://github.com/numenta/nupic/pull/2617 is merged.

### CODE REVIEW

**Comment by vitaly-krugl** [comment]
CC @lscheinkman, @marionleborgne, @scottpurdy

File: unicorn/backend/requirements.txt [hunk 1]
```diff
@@ -1,11 +1,8 @@
 validictory==0.9.1
 
 # NuPIC HTM
-nupic==0.3.0
-
-# Other Numenta projects
-htmengine>=0.0.0
```
Comment (oxtopus) [inline unicorn/backend/requirements.txt]: :+1:

File: unicorn/backend/requirements.txt [hunk 2]
```diff
@@ -1,11 +1,8 @@
...
-nta.utils>=0.0.0
+nupic==0.3.4
```
Comment (oxtopus) [inline unicorn/backend/requirements.txt]: CI won't pass until 0.3.4 is released.

**Comment by oxtopus** [comment]
LGTM :+1:

**Pull request merged**